.gitignore

```diff
@@ -1,8 +1,9 @@
-drivers
-www
-data
-ssl
-.env.*
 *.swp
+.env.*
 .htpasswd*
+.logs
+data
 docker-compose.yml
```

Comment:
This probably shouldn't be ignored.

Reply:
This is ignored as a result of the current mutation of the docker-compose. I think once the docker-compose.override.yml method is added, we should default track it.

I’ve been directly adding my changes for now.